src/views/Signup.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import { commitMutation } from 'react-relay'
+import environment from '../config/relay'
+import { Redirect } from "react-router";
+import graphql from 'babel-plugin-relay/macro'
+
+import {Form, Button, Container} from "react-bootstrap";
+
+const mutation = graphql`
+    mutation SignUpMutation($input: UserCreateInput!){
+        userCreate(input: $input){
+            user {
+                id
+            }
+        }
+    }
+`
+
+export default function SignUp() {
+    const [username, setUsername] = useState("");
```

Comment:
use single quotes `''` to match Standard JS syntax.

Reply:
got it. bad habit sorry haha